app/javascript/app/pages/climate-goals/overview/overview-component.jsx

```diff
@@ -1,15 +1,196 @@
+              ? (
+                <p
+                  className={styles.targetText}
+                  dangerouslySetInnerHTML={{ __html: values[type][0].value }}
+                />
+)
+              : renderNoContent()
+          }
+        </div>
+      </Card>
+    );
+
+    return (
+      <div className="grid-column-item">
+        <div className={styles.subtitles}>
+          {renderSubtitle(
+            t('pages.climate-goals.overview.mitigation-contribution')
+          )}
+        </div>
+        <div className={styles.cards}>
+          {renderGhgCard()}
+          {
+            nonGhgMitigationCards &&
+              nonGhgMitigationCards.map(
+                card => renderCard(card.title, card.type)
+              )
+          }
+        </div>
+        <div className={cx(styles.subtitles, styles.adaptationSubtitles)}>
+          {renderSubtitle(
+            t('pages.climate-goals.overview.adaptation-contribution')
+          )}
+        </div>
+        <div className={cx(styles.adaptationList, styles.adaptation)}>
+          <Card
+            reverse
+            theme={{
+              card: styles.card,
+              contentContainer: styles.contentContainer,
+              data: styles.adaptationData
+            }}
+            title={t('pages.climate-goals.overview.card-adaptation-action')}
+          >
+            <div className={styles.cardContent}>
+              {
+                sectors.length ? (
+                  <ul className={styles.list}>
+                    {sectors.map(sector => (
+                      <li key={sector} className={styles.listItem}>
+                        {sector}
+                      </li>
+                    ))}
+                  </ul>
+) : renderNoContent()
+              }
+            </div>
+          </Card>
+        </div>
+      </div>
+    );
+  };
+
   render() {
-    return <p className={styles.text}>The overview section</p>;
+    const { sectors, values, t } = this.props;
+    const hasSectors = values && sectors;
+
+    const description = hasSectors && (
+    <p
+      className={styles.descriptionContainer}
+          /* eslint-disable-next-line react/no-danger */
+      dangerouslySetInnerHTML={{
+            __html: get(values, 'indc_summary[0].value')
+          }}
+    />
+      );
+
+    return (
+      <React.Fragment>
+        <div className={styles.page}>
+          <div className={styles.sectionHeader}>
+            <SectionTitle
+              title={t('pages.climate-goals.overview.overview-title')}
+            />
+            <Button
+              onClick={this.handleBtnClick}
+              theme={{ button: cx(button.primary, styles.button) }}
+            >
+              <span
+                className={styles.buttonText}
+                dangerouslySetInnerHTML={{
+                  __html: t('pages.climate-goals.overview.button-title')
+                }}
+              />
+              <Icon icon={openInNewIcon} />
```

Comment:
This is kind of big. I created a theme for the icon button in India:
/app/javascript/app/styles/themes/icon.scss

Reply:
🙆